renovate.json

```diff
@@ -3,9 +3,13 @@
   "extends": [
     "config:recommended",
     ":configMigration",
-    ":gitSignOff"
+    ":gitSignOff",
+    ":automergeMinor"
+  ],
+  "gitAuthor": "Renovate Bot <renovate@whitesourcesoftware.com>",
```

Comment:
I think the author has to have the name and email of the k0s-bot account but it's broken anyway, so let's see if this actually works and fix it later if it doesn't.